src/identities.js

```diff
@@ -0,0 +1,95 @@
+'use strict'
+const Identity = require('./identity')
+const OrbitDBIdentityProvider = require('./orbit-db-identity-provider')
+const Keystore = require('orbit-db-keystore')
+const type = 'orbitdb'
+const keypath = './orbitdb/identity/keys'
+let supportedTypes = {
+  'orbitdb': OrbitDBIdentityProvider
+}
+
+const getHandlerFor = (type) => {
+  if (!Identities.isSupported(type)) {
+    throw new Error(`IdentityProvider type '${type}' is not supported`)
+  }
+  return supportedTypes[type]
+}
+
+class Identities {
+  constructor (options = {}) {
+    this._keystore = options.keystore || Keystore.create(options.keypath || keypath)
```

Comment:
This whole file is looking _snazzy_! 👍 Let's do the `keypath` renaming as discussed elsewhere and this should be all good.